client/app/bundles/shared/components/Switch.scss

```diff
@@ -0,0 +1,64 @@
+@import "_legacy.scss";
+@import "_base.scss";
+
+.switch {
+  position: relative;
+  display: inline-block;
+  width: 119px;
+  height: 49px;
+}
+
+.switch input {
+  position: absolute;
+  top: -99999px;
+  left: -99999px;
+}
```

Comment:
Ahh, while this works, I think you can get away with just using `display: none;`!